inst/include/Rcpp/Nullable.h

```diff
@@ -76,9 +76,8 @@ namespace Rcpp {
          *
          * @throw 'not initialized' if object has not been set
          */
-        inline operator SEXP() {
-            checkIfSet();
```

Comment:
I can see that.  Let me rephrase:  Why did you make that change? What is the reasoning?

I see no relation in that and the `const` change.  So why mess with it?